ibis/expr/types.py

```diff
@@ -529,11 +537,15 @@ def name(self, name):
 
 class ScalarExpr(ValueExpr):
 
-    pass
+    def _type_display(self):
+        return str(self.type())
 
 
 class ArrayExpr(ValueExpr):
 
+    def _type_display(self):
+        return 'array({0!s})'.format(self.type())
```

Comment:
Ah, it's in the setup.py metadata but we should drop 2.6, especially as it's not supported in conda-forge